src/connect.tsx

```diff
@@ -43,9 +44,9 @@ export declare interface ConnectOptions {
   mapStateToProps?: IMapStateToProps;
   mapDispatchToProps?: IMapDispatchToProps;
   options?: IConnectOptions;
-  mergeProps?(stateProps: any, dispatchProps: any, ownProps: any): any;
-  mapQueriesToProps?(opts: MapQueriesToPropsOptions): any; // WatchQueryHandle
-  mapMutationsToProps?(opts: MapMutationsToPropsOptions): any; // Mutation Handle
+  mergeProps?(stateProps: Object, dispatchProps: Object, ownProps: Object): Object;
+  mapQueriesToProps?(opts: MapQueriesToPropsOptions): Object; // WatchQueryHandle
+  mapMutationsToProps?(opts: MapQueriesToPropsOptions): Object; // Mutation Handle
```

Comment:
Why is this now `MapQueriesToPropsOptions` instead of `MapMutationsToPropsOptions`?